content/influxdb/cloud/organizations/buckets/bucket-schema/_index.md

```diff
@@ -134,7 +134,7 @@ Use the [`extended-output` flag](/influxdb/cloud/reference/cli/influx/bucket-sch
       --columns-file sensor.ndjson
     ```
 
-### Errors
+### Troubleshoot errors
 
 #### Not permitted by schema
```

Comment:
```suggestion
Troubleshoot and resolve the following bucket schema errors:
- [Not permitted by schema](#not-permitted-by-schema)
- [No measurement schemas](#no-measurement-schemas)
- [Failed to create measurement](#failed-to-create-measurement)

#### Not permitted by schema
```

Reply:
Is this list comprehensive? If so, maybe we could say The following error may occur when validating writes against an explicit bucket schema...or something along those lines